README.md

```diff
@@ -1,67 +1,69 @@
 - **Permissioned Ethereum Virtual Machine:** This virtual machine is built to observe the Ethereum operation code specification and additionally asserts the correct permissions have been granted. Permissioning is enforced through secure native functions and underlies all smart contract code. An arbitrary but finite amount of gas is handed out for every execution to ensure a finite execution duration - “You don’t need money to play, when you have permission to play”.
-- **Application Binary Interface (ABI):** transactions need to be formulated in a binary format that can be processed by the blockchain node.  Currently tooling provides functionality to compile, deploy and link solidity smart contracts and formulate transactions to call smart contracts on the chain.  For proof-of-concept purposes we provide a monax-contracts.js library that automatically mirrors the smart contracts deployed on the chain and to develop middleware solutions against the blockchain network.  Future work on the light client will be aware of the ABI to natively translate calls on the API into signed transactions that can be broadcast on the network.
+- **Application Binary Interface (ABI):** transactions need to be formulated in a binary format that can be processed by the blockchain node. Current tooling provides functionality to compile, deploy and link solidity smart contracts and formulate transactions to call smart contracts on the chain.
 - **API Gateway:** Burrow exposes REST and JSON-RPC endpoints to interact with the blockchain network and the application state through broadcasting transactions, or querying the current state of the application. Websockets allow to subscribe to events, which is particularly valuable as the consensus engine and smart contract application can give unambiguously finalised results to transactions within one blocktime of about one second.
 
 ## Installation
 
-`burrow` is intended to be used by the `monax chains` command via [monax](https://monax.io/docs). Available commands such as `make | start | stop | logs | inspect | update` are used for chain lifecycle management.
-
-### For Developers
-Dependency management for Burrow is managed with [glide](github.com/Masterminds/glide), and you can build Burrow from source by following
 
-- [Install go](https://golang.org/doc/install)
+- [Install go](https://golang.org/doc/install) and have `$GOPATH` set
 - Ensure you have `gmp` installed (`sudo apt-get install libgmp3-dev || brew install gmp`)
-- and execute following commands in a terminal:
-- `go get github.com/Masterminds/glide`
-- `go get -d github.com/hyperledger/burrow`
-- `REPO=$($GOPATH/src/github.com/hyperledger/burrow)`
-- `cd $REPO && glide install`
```

Comment:
I was unable to `build` without this. Is `glide install` not required in the latest build process?

Reply:
`glide` has been replaced by the official go tool for dependency management (`dep`) on the #develop branch